visidata/loaders/toml.py

```diff
@@ -38,6 +31,13 @@ def reload(self):
         self.columns = []
         self.rows = []
 
+        try:
+            # Python 3.11+
+            import tomllib
```

Comment:
Oh, so weird! I understand now. `tomllib` is in stdlib in Python 3.11.

@saulpw Would it be better to have a `sys.version_info` check for the Python version, or to have this `try/except` for `tomllib`? My instinct is doing a version info check might be better, in case someone ever puts a dummy package on PyPI for `tomllib`. But maybe that worry is unfounded, they block some names.

Reply:
I believe `tomli` was (and still is) on PyPI before most of it was included in the standard library and was renamed to `tomllib`. For that reason I think a backport appearing is unlikely. I like the EAFP approach with `try` `except` personally :-)

Reply:
Yeah, and the thinking is that the `sys.version_info` check would be more likely to be buggy/fragile!